aisdc/attacks/likelihood_attack.py

```diff
@@ -248,8 +273,7 @@ def run_scenario_from_preds(  # pylint: disable = too-many-statements, too-many-
         # Train N_SHADOW_MODELS shadow models
         logger.info("Training shadow models")
         for model_idx in range(self.args.n_shadow_models):
-            if model_idx % 10 == 0:
-                logger.info("Trained %d models", model_idx)
+            logger.info("Trained %d models", model_idx + 1)
```

Comment:
this is going to lob out an info line for every model. I think that's too much output. Original was every ten?